organization/_posts/2018-04-05-coordination.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "HSF Weekly Meeting #132, 4 April, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+#### *Present/Contributors*: Graeme Stewart, Giulio Eulisse, Marco Clemencic, Marco Petric, Benedikt Hegner, Pere Mato, Enric Tejedor, Michel Jouvin, David Crooks, Torre Wenaus, David Lange, Eduardo Rodrigues, Simone Campana, Helge Meinhard, Andrea Valassi, Daniel Elvira, Liz Sexton-Kennedy, Martin Ritter, Robert (K?), Sudhir Malik, John (H?)
+
+News, general matters
+=====================
+-   Call for papers to [UK Research Software Engineers 2018](http://rse.ac.uk/conf2018/) was circulated to lists.
+    -   A few people are interested in working on an HSF paper, Graeme
+        will follow up.
+-   2018 CERN School of Computing (CSC 2018) applications open,
+    [https://indico.cern.ch/e/CSC-2018](https://indico.cern.ch/e/CSC-2018).
+-   CERN OpenLab organising a 2 day workshop on Quantum Computing, w/b 5
+    November.
+
+HSF/WLCG Workshop Follow-up
+===========================
+-   Gather important items that we should be tracking after the workshop
+    (see workshop [live
+    notes](https://docs.google.com/document/d/1QSkvwRK_2HENuxYXcs9Op1dTUK824KddQ1Tfan-P0WU/edit?usp=sharing))
+    -   ### General Points
+        -   We need a newsletter!
+            -   Graeme to follow up
+        -   Benedikt will give a WS summary in next week's EP-SFT meeting.
+    -   ### Common Data Management and Data Lakes
+        -   WLCG lead here
```

Comment:
Not sure whether you care too much but there are several items throughout the minutes which do not finish with a dot.

Reply:
Not too much, but I put most of them in the last commit.